src/lib.rs

```diff
@@ -1,5 +1,9 @@
 //! Parser for bitmap fonts
 
+#[cfg(feature = "serde")]
+#[macro_use]
```

Comment:
Is macro_use still used?

Reply:
With the `master` branch being 2018 edition now I think this can be remove this. Will push a commit.